app/javascript/packs/actions/options.js

```diff
@@ -0,0 +1,62 @@
+import {
+  FETCH_IN_PROGRESS,
+  OPTION_FETCH_IN_PROGRESS,
+  OPTION_SAVE_IN_PROGRESS,
+  PRE_FETCH_SAVE,
+  CREATE_UNSAVED_OPTION,
+  FETCH_FAILURE,
+  FETCH_SUCCESS,
+  OPTION_FETCH_FAILURE,
+  OPTION_FETCH_SUCCESS,
+  REMOVE_OPTION
+} from '../constants';
+
+export function optionFetchInProgress(option) {
+  return {
+    type: OPTION_FETCH_IN_PROGRESS,
+    status: FETCH_IN_PROGRESS,
+    fetching: true,
```

Comment:
Are you using Kevin's `Status` component or something else to display fetch status? We decided today that having `fetching: true` and `status: FETCH_IN_PROGRESS` are redundant, in favor of having the more descriptive/flexible `status` property. Let me know if this has some other use that I'm not thinking of.